developing-packages.md

```diff
@@ -220,4 +220,48 @@ Pod::Spec.new do |s|
   s.dependency 'url_launcher'
 ```
 You can now `#import "UrlLauncherPlugin.h"` and access the `UrlLauncherPlugin` class in the source code
-at `hello/ios/Classes'.
+at `hello/ios/Classes`.
+
+### Conflict resolution
+
+It sometimes happens that your package or app depends on packages whose (transitive) dependencies
+are in conflict. Suppose you want to use `some_package` and `other_package` in package `hello`,
+but both of these depend on `url_launcher` in different versions that cannot be reconciled
+automatically. Unless the two packages actually use `url_launcher` in incompatible ways, this
+can be dealt with by adding a dependency override declaration to the `pubspec.yaml` file in
+`hello`, forcing the use of a particular version:
+
+In `hello/pubspec.yaml`:
+```yaml
+dependencies:
+  some_package:
+  other_package:
+dependency_overrides:
+  url_launcher: '0.4.2'
+```
+
+If the conflicting dependency is not itself a package, but an Android-specific library like `guava`,
+the dependency override declaration must be added to Gradle build logic instead.
+
+In `hello/android/build.gradle`:
+```groovy
+configurations.all {
+    resolutionStrategy { 
+        force 'com.google.guava:guava:23.0-android'
+    }
+}
+```
+
+Cocoapods does not currently offer override functionality for iOS-specific dependencies. But conflict
+resolution by override should be used only as a last resort anyway. Developers generally rely on package
+authors making good use of version _ranges_ rather than specific versions, e.g.:
```

Comment:
The text about last resort and using ranges seems like it applies to the other two as well and should be put earlier and made more generic, leaving only "Cocoapods does not currently offer override functionality for iOS-specific dependencies.", maybe saying "Please contact your package developer and ask them to use ranges instead" or something.